settings.py

```diff
@@ -24,5 +24,9 @@
 USE_TZ = False
```

Comment:
The `USE_TZ` setting is currently set to `False`. Consider setting it to `True` to handle datetime objects correctly across different timezones.